galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/src/main/java/dev/galasa/cicsts/ICicsRegion.java

```diff
@@ -52,6 +52,7 @@ public interface ICicsRegion {
     ICemt cemt() throws CicstsManagerException;    
     ICeda ceda() throws CicstsManagerException;    
     ICeci ceci() throws CicstsManagerException;
+    ITsqFactory getTSQFactory() throws CicstsManagerException;

```

Comment:
Cases need to be consistent ? `ITsqFactory getTsqFactory()` or `ITSQFactory getTSQFactory()` I guess I don't care which one, but using both seems odd.

Reply:
Updated to ITsqFactory and getTsqFactory().